python/cugraph/cugraph/dask/__init__.py

```diff
@@ -11,10 +11,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from cugraph.utilities.api_tools import experimental_warning_wrapper
+
 from .link_analysis.pagerank import pagerank
+from .link_analysis.hits import hits
 from .traversal.bfs import bfs
 from .traversal.sssp import sssp
 from .common.read_utils import get_chunksize
 from .community.louvain import louvain
 from .centrality.katz_centrality import katz_centrality
 from .components.connectivity import weakly_connected_components
+
+from .sampling.neighborhood_sampling import EXPERIMENTAL__uniform_neighborhood
+uniform_neighborhood = experimental_warning_wrapper(EXPERIMENTAL__uniform_neighborhood)
```

Comment:
We should move this to the `experimental` namespace, so this would go in `cugraph.experimental.dask`

I think we should also call it `uniform_neighborhood_sampling` since the `sampling` namespace isn't always used (as seen here).